dartsim/src/EntityManagement_TEST.cc

```diff
@@ -62,6 +62,12 @@ TEST(EntityManagement_TEST, ConstructEmptyWorld)
   EXPECT_EQ(world, model->GetWorld());
   EXPECT_NE(model, world->ConstructEmptyModel("dummy"));
 
+  auto nestedModel = model->ConstructEmptyModel("empty nested model");
+  ASSERT_NE(nullptr, nestedModel);
+  EXPECT_EQ("empty nested model", nestedModel->GetName());
+  EXPECT_EQ(world, nestedModel->GetWorld());
+  EXPECT_NE(nestedModel, nestedModel->ConstructEmptyModel("dummy"));
```

Comment:
It looks like we are testing `ConstructEmptyModel` here, but that was already done above. Did you mean to test `ConstructEmptyNestedModel` instead?

Reply:
Since `ConstructEmptyModel` was being called on a `Model` entity, it was actually calling `ConstructEmptyNestedModel` on the physics-plugin side. But to avoid confusion and per @mxgrey's [suggestion](https://github.com/ignitionrobotics/ign-physics/pull/228#discussion_r599303356), I have changed the user facing function to  `ConstructEmptyNestedModel` in https://github.com/ignitionrobotics/ign-physics/pull/228/commits/d137bba029a5cea24d22ee5458ef32b740e32fcf